app/docs/dev.md

```diff
@@ -1,29 +1,207 @@
-# YourFunctionName
-*Brief description of what this evaluation function does, from the developer perspective*
+# buckinghamPiTheorem
+
+Checks that the set of quantities in the response matches the set of quantities in the sense given by the Buckingham Pi theorem.
+
+There are three different ways of supplying this function with the necessary information.
+1. In the answer, provide an example set of groups as a comma separated list. When used this way the function assumes that the given list is correct and contains at least the minimum number of groups.
+2. In the `quantities` parameter, supply a list of what the dimensions for each quantity is and set answer to `-`. The function will then compute a list of sufficiently many independent dimensionless quantities and compare to the response.
+3. In the `quantities` parameter, supply a list of what the dimensions for each quantity is and in the answer, supply a list of groups as in the first option. The function will then check that the supplied answer is dimensionless and has a sufficient number of independent groups before comparing it to the response.
```

Comment:
Suggest:
3. Fill in the answer as per (1), and the quantities as per (2). The function will then check ... etc.
[It's not clear what will happen after the check?]

Reply:
Changed according to suggestions and clarified what happens after check